wdl/GatherSampleEvidence.wdl

```diff
@@ -76,6 +76,11 @@ workflow GatherSampleEvidence {
     File? baseline_melt_vcf
     File? baseline_scramble_vcf
 
+
+    # Localize reads parameters
+    # set to true on default, skips localize_reads if set to false
+    Boolean run_localize_reads = true
```

Comment:
I would move this parameter up after revise_base to better match the organization of parameters in this workflow - this new input pertains to actions on the CRAM file so it fits best towards the beginning